src/sqlalchemy_cratedb/compiler.py

```diff
@@ -323,3 +324,44 @@ def for_update_clause(self, select, **kw):
         warnings.warn("CrateDB does not support the 'INSERT ... FOR UPDATE' clause, "
                       "it will be omitted when generating SQL statements.")
         return ''
+
+
+CRATEDB_RESERVED_WORDS = \
```

Comment:
I've exluded the ones already marked as "reserved" in: https://www.postgresql.org/docs/current/sql-keywords-appendix.html
and the list is:
```
ADD, ALTER, BETWEEN, BY, CALLED, COSTS, DELETE, DENY, DIRECTORY, DROP, ESCAPE, EXISTS, EXTRACT, FIRST, FUNCTION, IF, INDEX, INPUT, INSERT, LAST, MATCH, NULLS, OBJECT, PERSISTENT, RECURSIVE, RESET, RETURNS, REVOKE, SET, STRATIFY, TRANSIENT, TRY_CAST, UNBOUNDED, UPDATE
```

Reply:
I know we concatenate them to a set, so it doesn't matter, maybe it's even better to list here all the cratedb reserved keywords, I'll leave it up to you.

Reply:
That's much shorter. I've updated the list correspondingly. Thanks.